### ISSUE TITLE
Class can be throwable by extending Error

### ISSUE THREAD

**Issue by AndrolGenhald** [php/doc-en#1647]
https://www.php.net/manual/en/class.throwable.php

> PHP classes cannot implement the Throwable interface directly, and must instead extend [Exception](https://www.php.net/manual/en/class.exception.php).

[`Error` can be extended](https://3v4l.org/QQHoN) now to create a throwable class as well.

**Comment by cmb69** [comment]
> [`Error` can be extended](https://3v4l.org/QQHoN) now to create a throwable class as well.

Right. However, the [`Error` documentation](https://www.php.net/manual/en/class.error.php) states:

> Error is the base class for all internal PHP errors.

So either users are not supposed to subclass `Error`, or that statement is misleading.

**Comment by AndrolGenhald** [comment]
I interpreted that as meaning all internal PHP errors will extend `Error`, but a specific instance of `Error` may or may not be an internal PHP error. If your interpretation is correct then I would have expected PHP to prevent extending `Error` in user code. If it's more of a "SHOULD" requirement rather than a "MUST" then that should be documented explicitly.
